performance/SwiftUIPerformanceQuickstart/PerformanceExample/Shared/Process.swift

```diff
@@ -177,7 +206,32 @@ class Process: ObservableObject {
     updateStatus(to: .success)
   }
 
-  func uploadImage() {}
+  func uploadImage(quality: CGFloat = 0.5) {
```

Comment:
Lets call the parameter `compressionQuality`.

Reply:
Done ✅